tests/filesTest.js

```diff
@@ -1146,23 +1260,23 @@ describe('Main: Currently testing files management,', function () {
   describe('favorite, search file', function () {
     const fileId = 123456789
     const tagId = 6789
+    const file = `${testFolder}/${testFile}`
+    const newTagName = 'testSystemTag12345'
 
     it('checking method: favorite', async function () {
       const provider = createProvider(true, true)
```

Comment:
should pass now in oc10, can you set it to `const provider = createProvider(false, true)`